### DISCUSSION TITLE
Is it possible that the bot control user's voice input?

### DISCUSSION THREAD

**Discussion by richardlee-kr** [Rapptz/discord.py#7078] [Ideas]
Title doesn't mean mute voice or volume control.
For example, by command to bot, music from bot is played by user's voice like sound from game streaming.
Command is received by bot but the real play is in user's voice.
If possible, i don't need to make a new program for my plan.

Sorry for poor English.
If something is uncomprehend, comment please.

**Comment by pikaninja** [comment]
The discord api does not support playing sound for other users

**Comment by Robin5605** [comment]
No, you cannot use your bot to control other's voice inputs, if that's what you mean. You can play music or some other audio content through your bot into a voice channel, if you would prefer that instead.
